app/main/posts/modify/post-video.directive.js

```diff
@@ -0,0 +1,92 @@
+module.exports = PostVideo;
+
+PostVideo.$inject = [];
+
+function PostVideo() {
+    return {
+        restrict: 'E',
+        scope: {
+            videoUrl: '='
+        },
+        templateUrl: 'templates/main/posts/modify/video.html',
+        controller: PostVideoController
+    };
+}
+
+PostVideoController.$inject = [
+    '$scope',
+    '$sce',
+    'Util'
+];
+
+function PostVideoController(
+    $scope,
+    $sce,
+    Util
+) {
+    $scope.constructIframe = constructIframe;
+    // $scope.getVideoThumbnail = getVideoThumbnail;
+    activate();
+
+    function activate() {
+        // Is this dody? I dunno it seems dodgy
+        $scope.videoUrl = $sce.trustAsResourceUrl($scope.videoUrl);
```

Comment:
I'm not clear what this does.. but it seems like we should sanity check before saying X is trusted?